GEON-PPANG-iOS/Global/Utils/LayoutUtils.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  LayoutUtils.swift
+//  GEON-PPANG-iOS
+//
+//  Created by JEONGEUN KIM on 2023/09/04.
+//
+
+import UIKit
+
+final class LayoutUtils {
+    
+    class func emptyGroup() -> NSCollectionLayoutGroup {
+        let item = NSCollectionLayoutItem(layoutSize: .init(
+            widthDimension: .fractionalWidth(1),
+            heightDimension: .fractionalHeight(1)
+        ))
+        
+        let groupSize = NSCollectionLayoutSize(
+            widthDimension: .fractionalWidth(1),
+            heightDimension: .fractionalHeight(1)
+        )
+        
+        let group = NSCollectionLayoutGroup.horizontal(layoutSize: groupSize, subitems: [item])
+        return group
+    }
+    
+    class func section(_ exists: Bool, _ size: CGFloat? = 0) -> NSCollectionLayoutSection {
+        
+        let group = LayoutUtils.emptyGroup()
+        let section = NSCollectionLayoutSection(group: group)
+        
+        if exists {
+            section.boundarySupplementaryItems = [LayoutUtils.header(size ?? 0)]
+        }
+        
+        return section
+    }
```

Comment:
p1:
함수 명이 section 인 상태로 input 값들도 _ 로 설정해두면 사용할 때 이게 어떤 함수인지 알기 어려울 것 같아여
코드로는 emptyView 에 header 가 있을 때 없을 때 나타내는 section 같은데 ( 맞는지는 모르겠음 ㅎ )
이런거라면

```suggestion
    class func emptySection(withHeight height: CGFloat = 0, hasHeader: Bool) -> NSCollectionLayoutSection {
        
        let group = LayoutUtils.emptyGroup()
        let section = NSCollectionLayoutSection(group: group)
        
        if hasHeader {
            section.boundarySupplementaryItems = [LayoutUtils.header(height ?? 0)]
        }
        
        return section
    }
```

요로코롬 한다면 사용할 때에도

```swift
LayoutUtils.emptySection(withHeight: 100, hasHeader: false)
```

로 쓰면 되니까 더 잘 읽히지 않을까 싶어요 !

Reply:
d2d71fefd6f2937fb9604e96d094b6c117a133d0